torch/csrc/jit/passes/quantization.cpp

```diff
@@ -692,6 +695,149 @@ c10::optional<IValue> toTwoElementIntList(Value* v) {
   }
   return c10::nullopt;
 }
+
+script::Module findChildModule(
+    const script::Module& module,
+    const std::vector<std::string>& path) {
+  script::Module m = module;
+  for (size_t i = 0; i < path.size(); ++i) {
+    m = m.attr(path[i]).toModule();
+  }
+  return m;
+}
+
+std::string addChildModule(
+    script::Module module,
+    const script::Module& child_module,
+    const std::vector<std::string>& path) {
+  script::Module cur = module;
+  TORCH_INTERNAL_ASSERT(
+      path.size() > 0, "path must have at least one element.");
+  for (size_t i = 0; i < path.size() - 1; ++i) {
+    cur = module.attr(path[i]).toModule();
```

Comment:
This loop is similar to `findChildModule`. If we change `findChildModule` to return the containing module of the module specified by the path (or the parent of the child module we're currently returning), then we can reuse the same code in these two places.